code/src/index.css

```diff
@@ -5,9 +5,149 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
 }
 
-code {
+/* code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
+} */
+
+.header {
+  display: flex;
+  flex-direction: column;
+  font-size: 20px;
+  align-self: center;
+  width: 95%;
+}
+
+.header-text {
+  /* color: white; */
+  align-self: flex-start;
+  font-size: 18px;
+}
+
+.header-line {
+  border-top: solid 2px #3f3f3f;
+  width: 100%;
+  align-self: center;
+}
+
+h2,
+h3 {
+  font-size: 14px;
+  margin: 8px 0px 5px 10px;
+}
+
+h2:hover,
+.artist-name:hover {
+  text-decoration: underline;
+  color: white;
+}
+
+.break-word {
+  margin: unset;
+}
+
+.main-view {
+  margin: 10px;
+  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+}
+
+.album-section {
+  max-width: 1645px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin: auto;
+}
+
+.album-card {
+  margin: 10px 10px 20px 10px;
+  flex-direction: column;
+  align-items: center;
+  padding: 10px;
+  margin: 10px;
+}
+
+.images-container {
+  display: flex;
+  position: relative;
+}
+
+.heart-icon,
+.play-icon,
+.dots-icon {
+  filter: invert(100%);
+  width: 50px;
+  height: 50px;
 }
+
+.icons {
+  position: absolute;
+  flex-direction: row;
+  display: none;
+  gap: 50px;
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  justify-content: center;
+}
+
+.play-icon:hover {
+  transform: scale(1.5);
+}
+
+.album-img {
+  width: 100%;
+  height: auto;
+}
+
+.images-container:hover .icons {
+  display: flex;
+  background-color: rgba(0, 0, 0, 0.6);
+}
+
+.images-container:hover {}
+
+.album-name {
+  text-decoration: none;
+  color: white;
+}
+
+.artist-container {
```

Comment:
Minor thing, the artist name and underline turns white when you hover over it. It does not affect the functionality it is only if you want to show the same as the design brief.